File: core/src/main/java/com/web3auth/core/types/WhiteLabelData.kt
package com.web3auth.core.types

import androidx.annotation.Keep
import java.io.Serializable

@Keep
data class WhiteLabelData(
    @Keep var appName: String? = null,
    @Keep var appUrl: String? = null,
    @Keep var logoLight: String? = null,
    @Keep var logoDark: String? = null,
    @Keep var defaultLanguage: Language? = Language.EN,
    @Keep var mode: ThemeModes? = null,
    @Keep var useLogoLoader: Boolean? = false,
    @Keep var theme: HashMap<String, String?>? = null
) : Serializable

File: Assets/Plugins/Web3AuthSDK/Types/WhiteLabelData.cs
using System.Collections.Generic;

public class WhiteLabelData { 
    public string? appName { get; set; }
    public string? logoLight { get; set; }
    public string? logoDark { get; set; }
    public string? defaultLanguage { get; set; } = "en";
    public string? mode { get; set; } = "light";
    public Dictionary<string, string>? theme { get; set; }
    public string? appUrl { get; set; }
    public bool? useLogoLoader { get; set; } = false;
}